Formula/rust-mode.rb

```diff
@@ -20,7 +20,7 @@ def caveats; <<-EOS.undent
     Add the following to your init file:
 
     (autoload 'rust-mode "rust-mode" nil t)
-    (add-to-list 'auto-mode-alist '("\\.rs\\'" . rust-mode))
+    (add-to-list 'auto-mode-alist '("\\\\.rs\\\\'" . rust-mode))
```

Comment:
I've started removing caveats that aren't Homebrew-specific; since they just duplicate upstream docs and can get out-of-date without anyone realizing.  If this isn't Homebrew-specific, would you mind just removing the entire caveats?